binderhub/builder.py

```diff
@@ -391,7 +391,7 @@ async def get(self, provider_prefix, _unescaped_spec):
                     break
                 except HTTPClientError:
                     app_log.exception(
-                        "Tornado HTTP Timeout error: Failed to get image manifest for %s",
+                        "Failed to get image manifest for %s",
```

Comment:
This error log would occur for the 401. Timeout is not the only reason for this situation.